.gitattributes

```diff
@@ -14,4 +15,4 @@
 /test/** coverage-excluded=true
 /cmd/**/main.go coverage-excluded=true
 /pkg/broker/config/targets.pb.go coverage-excluded=true
-/pkg/broker/handler/pool/testing/** coverage-excluded=true
\ No newline at end of file
+/pkg/broker/handler/pool/testing/** coverage-excluded=true
```

Comment:
I think this line can be removed, since this directory doesn't exist anymore.

Reply:
done